vulnerabilities/tests/test_data/apache_kafka/jmh-test-01.txt

```diff
@@ -0,0 +1,84 @@
+{'aliases': ['CVE-2022-34917'], 'summary': 'This CVE identified a flaw where it allows the malicious unauthenticated clients to allocate large amounts of memory on brokers. This can lead to brokers hitting OutOfMemoryException and causing denial of service.', 'affected_packages': [{'package': {'type': 'maven', 'namespace': None, 'name': 'apache_kafka', 'version': None, 'qualifiers': None, 'subpath': None}, 'affected_version_range': 'vers:maven/>=2.8.0|<=2.8.1|!=2.8.2|>=3.0.0|<=3.0.1|!=3.0.2|>=3.1.0|<=3.1.1|!=3.1.2|>=3.2.0|<=3.2.1|!=3.2.3', 'fixed_version': None}], 'references': [{'reference_id': '', 'url': 'https://kafka.apache.org/cve-list', 'severities': []}, {'reference_id': 'CVE-2022-34917', 'url': 'https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2022-34917', 'severities': []}], 'date_published': None}
```

Comment:
What is this file?

Reply:
Leftover from earlier testing -- I just deleted this file.